supabase/migrations/20260816120150_org_member_read_billing_default.sql

```
@@ -0,0 +1,499 @@
+AS $$
+  SELECT COALESCE(array_agg(DISTINCT orgs.customer_id::text), '{}'::text[])
+  FROM public.orgs
+  WHERE orgs.customer_id IS NOT NULL
+    AND orgs.id = ANY(COALESCE(
+      (SELECT public.rbac_org_ids_for_permission(public.rbac_perm_org_read_billing())),
+      '{}'::uuid[]
+    ))
+$$;
+
+COMMENT ON FUNCTION public.readable_org_customer_ids() IS
+  'Customer IDs whose stripe_info rows are readable by the caller through org.read_billing.';
```

SQLFluff (4.2.2)

[error] Line is too long (93 > 80).

(LT05)